src/utils/parse.js

```diff
@@ -237,4 +237,4 @@ export const generateSampleTable = response => {
     runs,
     iterationParams,
   };
-};
+};
```

Comment:
You lost the newline at the end; it might not be necessary, but GitHub complains, and other tools might as well.